Sources/unitfrmpgndatabase.pas

```diff
@@ -0,0 +1,156 @@
+{
+  This file is part of Chess 256.
+
+  Copyright © 2016, 2018 Alexander Kernozhitsky <sh200105@mail.ru>
+
+  Chess 256 is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Chess 256 is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Chess 256.  If not, see <http://www.gnu.org/licenses/>.
+
+  Abstract:
+    This is a Chess PGN Database window.
+    This file Coded by Jim Kinsman <relipse@gmail.com>
+}
+unit unitFrmPgnDatabase;
+
+{$mode ObjFPC}{$H+}
+
+interface
+
+uses
+  Classes, SysUtils, Forms, Controls, Graphics, Dialogs, StdCtrls,
+  ExtCtrls, Grids, Menus, Types, BoardForms;
+
+type
+
+  { TfrmPGNDatabase }
+
+  TfrmPGNDatabase = class(TForm)
+    mniLoadGame: TMenuItem;
+    pnlDatabase: TPanel;
+    gridPGNDatabase: TStringGrid;
+    pmnPGNDatabase: TPopupMenu;
+    procedure gridPGNDatabaseDblClick(Sender: TObject);
+    procedure gridPGNDatabaseDrawCell(Sender: TObject; aCol, aRow: Integer;
+      aRect: TRect; aState: TGridDrawState);
+    procedure gridPGNDatabaseMouseMove(Sender: TObject; Shift: TShiftState; X,
+      Y: Integer);
+    procedure gridPGNDatabasePrepareCanvas(Sender: TObject; aCol,
+      aRow: Integer; aState: TGridDrawState);
+    procedure mniLoadGameClick(Sender: TObject);
+  private
+
+  public
+
+  end;
+
+var
+  frmPGNDatabase: TfrmPGNDatabase;
+  My_Grid_Highlight_BackGround_Color:Tcolor=ClGreen;
+  My_Grid_Highlight_Font_Color:Tcolor=clYellow;
+  My_Grid_Highlight_Font_Multiplier:Real=1.5;  // FOnt Multiplier for selected row
+  LastRow : Integer;
```

Comment:
If these are constants, please mark them as `const` and move to `implementation` section

If these are variables for something on the form, please make it `private` fields.